rust/benches/messages/update.rs

```diff
@@ -0,0 +1,94 @@
+#![allow(non_snake_case)]
+
+mod utils;
+
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
+use xaynet_core::message::{FromBytes, Update};
+
+pub fn parse_tiny(c: &mut Criterion) {
```

Comment:
it seems that all the `parse_...` functions could be generated by a macro. on the other hand it's just around 100 lines of code, so maybe not needed. up to you.

Reply:
Agree it would be worth a macro, but I didn't want to put that much effort into it since it's just a benchmark.